src/lib/viewers/doc/DocFindBar.js

```diff
@@ -137,12 +138,19 @@ class DocFindBar extends EventEmitter {
      * @return {void}
      */
     dispatchFindEvent(type, findPrev) {
-        this.findController.executeCommand(type, {
-            query: this.findFieldEl.value,
-            phraseSearch: true, // true by default
-            highlightAll: true, // true by default
-            findPrevious: findPrev,
-        });
+        return Browser.isIE()
+            ? this.findController.executeCommand(type, {
+                  query: this.findFieldEl.value,
+                  phraseSearch: true, // true by default
+                  highlightAll: true, // true by default
+                  findPrevious: findPrev,
+              })
+            : this.findController._eventBus.dispatch(type, {
```

Comment:
Can we use `this.eventBus` rather than accessing the semi-private member on `findController`?

Reply:
Did you mean doing something like `this.eventBus.dispatch...`?